modules/service/src/main/scala/ch/epfl/bluebrain/nexus/commons/service/retryer/TaskRetry.scala

```diff
@@ -6,7 +6,7 @@ import monix.execution.CancelableFuture
 import monix.execution.Scheduler.Implicits.global
 
 import scala.concurrent.Future
-import scala.concurrent.duration.{FiniteDuration, _}
+import scala.concurrent.duration._
 import scala.util.Random
 
 trait TaskRetry {
```

Comment:
could you please rename this to RetryOps instead? I'd like to avoid exposing that we're using a task for now.